policy/modules/services/smartmon.te

```diff
@@ -143,7 +143,9 @@ corenet_tcp_connect_http_port(smartmon_update_drivedb_t)
 
 files_read_etc_files(smartmon_update_drivedb_t)
 
-fstools_exec(smartmon_update_drivedb_t)
+optional_policy(`
+	fstools_exec(smartmon_update_drivedb_t)
+')
```

Comment:
I expect @pebenito will want this moved lower in the file, with the other 'optional_policy' blocks.

Reply:
Done